common/lib/xmodule/xmodule/js/src/video/09_video_caption.js

```diff
@@ -1017,6 +1052,98 @@
                 }
             },
 
+            handleCaptioningCookie: function() {
```

Comment:
There was already cookie functions in this file and I just copied and modified them, so I'm assuming the jQuery plugin is already loaded. It stores the cookie properly.